package.json

```diff
@@ -23,9 +23,7 @@
     "moment": "^2.10.6",
     "numeral": "^1.5.3",
     "react-addons-shallow-compare": "^0.14.7",
-    "recompose": "^0.15.1"
-  },
-  "peerDependencies": {
+    "recompose": "^0.15.1",
     "react": "^0.14.2",
```

Comment:
also yeah i'm aware of that note re: peerDependencies - it's been that way for awhile, but they still need to be peerDependencies. The only way this would cause an issue is if someone tried to install Reactochart in a project that didn't use React (which would be silly) - in which case they would get a warning upon `npm install`